### PR TITLE
added 7-1 sass folder structure

### PR DESCRIPTION
**Pull request by JustinRoman**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
         "lodash": "^4.17.13",
         "resolve-url-loader": "^3.1.0",
         "sass": "^1.15.2",
-        "sass-loader": "^8.0.0"
+        "sass-loader": "^8.0.0",
+        "vue-template-compiler": "^2.6.11"
```
Comment (tali-soupBrain) [inline package.json:20]: why do you need vue template compiler ?

File: public/css/app.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
+html,
```
Comment (tali-soupBrain) [inline public/css/app.css:1]: i think you done a reset file ? if yes recall it reset. also why is it css if we are with sass ?

File: resources/sass/app.scss [hunk 3]
```diff
@@ -1 +1,18 @@
-//
+// Base
+@import 'base/reset';
+@import 'base/typography';
```
Comment (tali-soupBrain) [inline resources/sass/app.scss:3]: verify well that the structure of folders respect the 7.1

File: package.json (hunk 1 above)
Comment (JustinRoman) [reply package.json:20]: its in the package.json. its what's added when i run npm command.

File: public/css/app.css (hunk 2 above)
Comment (JustinRoman) [reply public/css/app.css:1]: its compiled scss

File: resources/sass/app.scss (hunk 3 above)
Comment (JustinRoman) [reply resources/sass/app.scss:3]: it does. we just dont have vendors that's why there's only 6

File: package.json (hunk 1 above)
Comment (tali-soupBrain) [reply package.json:20]: Noted.

File: public/css/app.css (hunk 2 above)
Comment (tali-soupBrain) [reply public/css/app.css:1]: noted.

File: resources/sass/app.scss (hunk 3 above)
Comment (tali-soupBrain) [reply resources/sass/app.scss:3]: Noted.

**Pull request merged**